components/Micrio.vue

```diff
@@ -51,6 +73,14 @@ onMounted(() => {
 
     micrio.state.tour.subscribe((tour) => {
       if (tour) {
+        const markerTour = tour as
+          | Models.ImageCultureData.MarkerTour
+          | undefined;
+
+        if (markerTour) {
+          tourSteps.value = markerTour.steps;
+        }
```

Comment:
You already have this checked for falsey.
```suggestion
        const markerTour = tour as
          | Models.ImageCultureData.MarkerTour;

        tourSteps.value = markerTour.steps;
```